fb.d.ts

```diff
@@ -0,0 +1,61 @@
+import Promise = require("any-promise");
```

Comment:
You can remove this line. `Promise` can be used directly without importing it as it is part of ES6.